source/git-util.js

```diff
@@ -29,6 +29,21 @@ const firstCommit = async () => {
 	return stdout;
 };
 
+exports.tagBeforeCurrentOrFirstCommit = async () => {
+	const {stdout} = await execa('git', ['tag']);
+	const tags = stdout.split('\n');
+
+	if (tags.length === 0) {
+		return null;
```

Comment:
```suggestion
		return;
```

I prefer not to use `null` in my code.